src/DiffEqGPU.jl

```diff
@@ -1164,23 +1175,23 @@ end
-    @cuprintf "%2.2f %2.2f %2.2f\n%2.2f %2.2f %2.2f\n%2.2f %2.2f %2.2f" A[ii[1, 1]] A[ii[1,
-                                                                                         2]] A[ii[1,
-                                                                                                  3]] A[ii[2,
-                                                                                                           1]] A[ii[2,
-                                                                                                                    2]] A[ii[2,
-                                                                                                                             3]] A[ii[3,
-                                                                                                                                      1]] A[ii[3,
-                                                                                                                                               2]] A[ii[3,
-                                                                                                                                                        3]]
-end
+# function __printjac(A, ii)
+#     @cuprintf "[%d, %d]\n" ii.offset1 ii.stride2
+#     @cuprintf "%d %d %d\n%d %d %d\n%d %d %d\n" ii[1, 1] ii[1, 2] ii[1, 3] ii[2, 1] ii[2, 2] ii[2,
+#                                                                                                3] ii[3,
+#                                                                                                      1] ii[3,
+#                                                                                                            2] ii[3,
+#                                                                                                                  3]
+#     @cuprintf "%2.2f %2.2f %2.2f\n%2.2f %2.2f %2.2f\n%2.2f %2.2f %2.2f" A[ii[1, 1]] A[ii[1,
+#                                                                                          2]] A[ii[1,
+#                                                                                                   3]] A[ii[2,
+#                                                                                                            1]] A[ii[2,
+#                                                                                                                     2]] A[ii[2,
+#                                                                                                                              3]] A[ii[3,
+#                                                                                                                                       1]] A[ii[3,
+#                                                                                                                                                2]] A[ii[3,
+#                                                                                                                                                         3]]
+# end
```

Comment:
?

Reply:
Ah since CUDA is a weak deps now, the library won’t compile with this, due to the usage of `@cuprintf` (This function isn’t used anywhere as well though in the code)

Reply:
then just remove it?